valispace/__init__.py

```diff
@@ -169,6 +187,43 @@ def update_vali(self, name=None, id=None, formula=None, data={}):
 
 		return result
 
+	def post_data(self, type=None, data={}):
+		# Post new component/vali/textvali/tags with the input data
```

Comment:
Maybe put comment describing function in """ xxx """ style (also for other functions)? See https://stackoverflow.com/questions/2357230/what-is-the-proper-way-to-comment-functions-in-python